.github/workflows/CI.yml

```diff
@@ -39,6 +39,14 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
+      - name: "Compat fix for Julia < v1.3.0"
+        if: ${{ matrix.julia-version == '1.0' }}
+        run: |
+          using Pkg
+          Pkg.add([
+            PackageSpec(name="AbstractFFTs", version="0.5"),
+          ])
+        shell: julia --project=. --startup=no --color=yes {0}
```

Comment:
Actually, since this CI file uses `version` instead of `julia-version`, this stage will never be executed.

Reply:
I did not know that. Should I remove CVC and try with ImageCore only with the changes you have suggested?

Reply:
I don't know, but the purpose of this stage is to assist the old Pkg version to figure out an existing solution. If we get a green check without this stage executed, then it is not needed.